playbooks/roles/load-balancer-v2/files/haproxy.cfg.ctmpl

```diff
@@ -114,7 +103,6 @@ backend be-{{ $domain_slug }}
     http-request set-header Authorization 'Basic {{ $http_auth_info_base64 }}' unless has-authorization
     option httpchk /heartbeat
     {{- range $active_app_servers }}
-    server {{ $server_name }} {{ .public_ip }}:80 cookie {{ $server_name }} {{ if $health_check }}check{{ end }}
+    server appserver-{{ .id }} {{ .public_ip }}:80 cookie appserver-{{ .id }} {{ if $health_check }}check{{ end }}
```

Comment:
Where does the `.id` come from?

Reply:
Each dict in the `active_app_servers` list has an ID (equal to the appserver ID in Ocim) and the public IP.

```bash
root@haproxy-stage-1:~# consul kv get ocim/instances/343/active_app_servers
[{"public_ip": "147.135.195.136", "id": 1010}]
```